packages/web/src/components/upload/AudioQuality.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex } from '@audius/harmony'
+
+import { Text } from 'components/typography'
+
+import styles from './AudioQuality.module.css'
+
+const messages = {
+  lossless: 'Provide FLAC, WAV, ALAC, or AIFF for highest audio quality.',
+  learnMore: 'Learn more about lossless HD.'
+}
+
+type AudioQualityProps = {
+  className?: string
```

Comment:
nit: i think we're slightly preferring `css` prop now, but maybe a bit trickier to wrangle it. up to you!